package.json

```diff
@@ -26,6 +26,10 @@
     ".": {
       "import": "./lib/mjs/index.js",
       "require": "./lib/cjs/index.js"
+    },
+    "./commonJs": {
```

Comment:
I could swear I left a comment here but it somehow disappeared. 

Do you have a preference `cjs` vs `commonJs`? If not, can we do `cjs`?